netstandard/pkg/NETStandard.Library.pkgproj

```diff
@@ -20,9 +20,6 @@
     <ProjectReference Include="..\ref\netstandard.csproj" />
     <ProjectReference Include="shims\netstandard\dir.builds" />
     <ProjectReference Include="shims\netfx\dir.builds" />
-    <ProjectReference Include="..\src\netstandard.csproj">
```

Comment:
Looks to me like it has more configurations than just net461 so I didn't touch it.
https://github.com/dotnet/standard/blob/master/netstandard/src/netstandard.builds